index.js

```diff
@@ -227,12 +227,17 @@ assert.notImplement = (subject, methods) => {
 
 assert.objectContent = (obj, content) => {
   Object.keys(content).forEach(key => {
-    if (isObject(content[key])) {
-      assert.objectContent(obj[key], content[key]);
+    let actualValue = obj[key];
+    if (actualValue && isObject(content[key])) {
+      assert.objectContent(actualValue, content[key]);
       return;
     }
 
-    assert.equal(obj[key], content[key]);
+    if (Array.isArray(content[key]) && !content[key].length && actualValue === undefined) {
```

Comment:
> `&& actualValue === undefined`

No tests fail if this is removed - both here and later on. Either there's some missing unit test to be added or these checks aren't necessary and can be removed. Either way, could you check and make the right change please?